packages/mobile/src/home/RecentlyUsedDapps.tsx

```diff
@@ -0,0 +1,117 @@
+import { Colors } from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Spacing } from '@celo/react-components/styles/styles'
+import variables from '@celo/react-components/styles/variables'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, ScrollView, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { recentDappsSelector } from 'src/app/selectors'
+import { Dapp } from 'src/app/types'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import useSelector from 'src/redux/useSelector'
+
+interface Props {
+  onSelectDapp(dapp: Dapp): void
+}
+
+const DAPP_ICON_SIZE = 68
+
+function RecentlyUsedDapps({ onSelectDapp }: Props) {
+  const recentlyUsedDapps = useSelector(recentDappsSelector)
+  const { t } = useTranslation()
+
+  const onPressAllDapps = () => {
+    navigate(Screens.DAppsExplorerScreen)
+  }
+
+  if (!recentlyUsedDapps.length) {
+    return null
+  }
+
+  return (
+    <View style={styles.body} testID="RecentlyUsedDappsContainer">
+      <View style={[styles.titleContainer, styles.row]}>
+        <Text style={styles.title}>{t('recentlyUsedDapps')}</Text>
+        <TouchableOpacity style={styles.row} onPress={onPressAllDapps} testID="AllDapps">
```

Comment:
ah interesting, i didn't notice the subtle difference between the components.

i've updated to use `Touchable` but we should probably update this component at some point since `react-native-platform-touchable` is no longer maintained and they recommend using the button from `react-native-gesture-handler` instead